controllers/profiles/reconciler_dev.go

```diff
@@ -42,13 +48,22 @@ const (
 	//
 	// See: https://quarkus.io/guides/config-reference#application-properties-file
 	quarkusDevConfigMountPath    = "/home/kogito/serverless-workflow-project/src/main/resources"
+	camelDevConfigMountPath      = "/home/kogito/serverless-workflow-project/src/main/resources/routes"
+	genericDevConfigMountPath    = "/home/kogito/serverless-workflow-project/src/main/resources/generic"
+	openApiDevConfigMountPath    = "/home/kogito/serverless-workflow-project/src/main/resources/specs"
+	asyncApiDevConfigMountPath   = "/home/kogito/serverless-workflow-project/src/main/resources/templates" //@TODO
```

Comment:
could we make this a bit more generic ? like using a struc with prefix and mount path ?

Reply:
ok